source/windows/file.c

```diff
@@ -14,6 +14,10 @@
 #include <windows.h>
 
```

Comment:
I'd make a static s_is_non_empty(...) that checks for non-null and string len > 0